vcluster/configure/vcluster-yaml/control-plane/components/backing-store/etcd/embedded.mdx

```diff
@@ -13,14 +13,17 @@ import ProAdmonition from '@site/vcluster/_partials/admonitions/pro-admonition.m
 
 etcd is deployed with vCluster to reduce the overall footprint while still providing the advantages of etcd.
 
-You can migrate from an external etcd instance to the embedded one by configuring `migrateFromDeployedEtcd: true`.
+You can migrate from an external etcd instance to the embedded one by configuring `migrateFromDeployedEtcd: true` as shown below. However, by default, `enabled` is set to `false`, and `migrateFromDeployedEtcd` is also set to `false`.
```

Comment:
I'm not sure if you need any additional information. Did you test this out? 

But otherise, I'd change it to:

```
You can migrate from an external etcd instance to the embedded one by configuring `migrateFromDeployedEtcd: true`. You will also need to enable the embedded etcd in order to enable the migration. 
```